_63B/kbareset.md

```diff
@@ -0,0 +1,12 @@
+---
+question: B15
+title: Is it permissible to use sets of questions and answers stored by the subscriber for password reset?
+---
+Self-serve password reset requires authentication of the account owner (subscriber) in order to reset the password. Response to sets of knowledge-based questions is a form of knowledge-based authentication (KBA), and is not allowed to be used as described in [FAQ B07 above](#q-b07) and [SP 800-63-3 Section 4.3.1](https://pages.nist.gov/800-63-3/sp800-63-3.html#431-authenticators):
+
+> Knowledge-based authentication, where the claimant is prompted to answer questions that are presumably known only by the claimant, also does not constitute an acceptable secret for digital authentication.
+
+KBA for password reset would leave the account vulnerable to takeover.
+
+Alternative authenticators for password reset include lists of look-up secrets and out-of-band device authentication. See [NIST Special Publication 800-63B](https://pages.nist.gov/800-63-3/sp800-63b.html) for more details for these authentication processes.
+
```

Comment:
I think we should add here the notion of re-proofing the user as a means of account recovery. Go through the proofing process, assign new authenticators, revoke old authenticators, go on with life. It's drastic and heavyweight, but it's secure and uses existing mechanisms and patterns.

Reply:
This is probably necessary. Additional text -- " If alternative authenticators are not available for password reset, then the subscriber's identity will need to be re-proofed in order to gain account access. See NIST SP 800-63B section 6.1.2.3 Replacement of a Lost Authentication Factor for additional details."